api/src/main/java/com/velocitypowered/api/command/InvocableCommand.java

```diff
@@ -45,7 +45,6 @@
    *
    * @param invocation the invocation context
    * @return the tab complete suggestions
-   * @implSpec defaults to wrapping the value returned by {@link #suggest(CommandInvocation)}
```

Comment:
I recall this gave you a compile error, but this is a problem with the build tool, not the javadoc. See https://openjdk.java.net/jeps/8068562 and https://stackoverflow.com/a/55150289 for details. If we really need to remove it, move the contents to a new paragraph.